planet_explorer/gui/pe_tasking_dockwidget.py

```diff
@@ -135,7 +135,9 @@ def __init__(self, pt):
     def _link_clicked(self, url):
         if url.toString() == "dashboard":
             analytics_track(SKYSAT_TASK_CREATED)
-            url = f"https://www.planet.com/tasking/orders/new/#/geometry/{self.pt.asWkt()}"
+            url = (
+                f"https://www.planet.com/tasking/orders/new/?geometry={self.pt.asWkt()}"
```

Comment:
@Samweli I believe the latitude/longitude are flipped. 

In QGIS:
![Screenshot from 2022-11-09 07-44-07](https://user-images.githubusercontent.com/44065123/200875432-59bf70f3-0d7f-4139-84f6-39651777eba2.png)


In the tasking dashboard:
![Screenshot from 2022-11-09 07-44-15](https://user-images.githubusercontent.com/44065123/200875504-a43bd56d-6aa2-44ed-805c-7a9898fe74fe.png)

Reply:
yes you are right @john-dupuy , I have created another PR here https://github.com/planetlabs/qgis-planet-plugin/pull/72, thanks.